cwms-radar-model/src/main/java/mil/army/usace/hec/cwms/radar/client/model/Location.java

```diff
@@ -0,0 +1,366 @@
+package mil.army.usace.hec.cwms.radar.client.model;
```

Comment:
not related to this change

Reply:
Once PR6 is merged, i can do another rebase off main, which will remove this